src/modules/esl-anchornav/core/esl-anchornav.ts

```diff
@@ -0,0 +1,223 @@
+import {ExportNs} from '../../esl-utils/environment/export-ns';
+import {ESLBaseElement} from '../../esl-base-element/core';
+import {attr, decorate, listen, memoize, prop, ready} from '../../esl-utils/decorators';
+import {debounce, microtask} from '../../esl-utils/async';
+import {getViewportForEl} from '../../esl-utils/dom/scroll';
+import {ESLEventUtils, ESLIntersectionTarget} from '../../esl-event-listener/core';
+
+import type {DelegatedEvent, ESLIntersectionEvent} from '../../esl-event-listener/core';
+import type {ESLAnchorData, ESLAnchornavRender} from './esl-anchornav-types';
+
+/**
+ * ESLSAnchornav
+ * @author Dmytro Shovchko
+ *
+ * ESLAnchornav is a component that collects content anchors from the page and provides anchor navigation
+ */
+@ExportNs('Anchornav')
+export class ESLAnchornav extends ESLBaseElement {
+  public static override is = 'esl-anchornav';
+  public static _renderers: Map<string, ESLAnchornavRender> = new Map();
+
+  /** Default renderer for anchornav item */
+  public static defaultRenderer: ESLAnchornavRender = (data: ESLAnchorData) => `<a class="esl-anchornav-item" href="#${data.id}">${data.title}</a>`;
```

Comment:
Could be simplified. 
Use `default` as ommitable renderer key.

So then just will be enough to have default
```
ESLAnchornav.setRenderer(
   (data: ESLAnchorData) => `<a class="esl-anchornav-item" href="#${data.id}">${data.title}</a>`
);
```